sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -66,75 +66,74 @@
+        - pyamqp.authentication.SASLAnonymous
+        - pyamqp.authentication.SASLPlain
+        - pyamqp.authentication.SASTokenAuth
+        - pyamqp.authentication.JWTTokenAuth
      If no authentication is supplied, SASLAnnoymous will be used by default.
-    :type auth: ~uamqp.authentication.common.AMQPAuth
-    :param client_name: The name for the client, also known as the Container ID.
+    :paramtype auth: ~pyamqp.authentication
+    :keyword client_name: The name for the client, also known as the Container ID.
      If no name is provided, a random GUID will be used.
-    :type client_name: str or bytes
-    :param debug: Whether to turn on network trace logs. If `True`, trace logs
+    :paramtype client_name: str or bytes
+    :keyword debug: Whether to turn on network trace logs. If `True`, trace logs
      will be logged at INFO level. Default is `False`.
-    :type debug: bool
-    :param retry_policy: A policy for parsing errors on link, connection and message
+    :paramtype debug: bool
+    :keyword retry_policy: A policy for parsing errors on link, connection and message
      disposition to determine whether the error should be retryable.
-    :type retry_policy: ~uamqp.errors.RetryPolicy
-    :param keep_alive_interval: If set, a thread will be started to keep the connection
+    :paramtype retry_policy: ~pyamqp.error.RetryPolicy
+    :keyword keep_alive_interval: If set, a thread will be started to keep the connection
      alive during periods of user inactivity. The value will determine how long the
      thread will sleep (in seconds) between pinging the connection. If 0 or None, no
      thread will be started.
-    :type keep_alive_interval: int
-    :param max_frame_size: Maximum AMQP frame size. Default is 63488 bytes.
-    :type max_frame_size: int
-    :param channel_max: Maximum number of Session channels in the Connection.
-    :type channel_max: int
-    :param idle_timeout: Timeout in seconds after which the Connection will close
+    :paramtype keep_alive_interval: int
+    :keyword max_frame_size: Maximum AMQP frame size. Default is 63488 bytes.
+    :paramtype max_frame_size: int
+    :keyword channel_max: Maximum number of Session channels in the Connection.
+    :paramtype channel_max: int
+    :keyword idle_timeout: Timeout in seconds after which the Connection will close
      if there is no further activity.
-    :type idle_timeout: int
-    :param auth_timeout: Timeout in seconds for CBS authentication. Otherwise this value will be ignored.
+    :paramtype idle_timeout: int
+    :keyword auth_timeout: Timeout in seconds for CBS authentication. Otherwise this value will be ignored.
      Default value is 60s.
-    :type auth_timeout: int
-    :param properties: Connection properties.
-    :type properties: dict
-    :param remote_idle_timeout_empty_frame_send_ratio: Ratio of empty frames to
+    :paramtype auth_timeout: int
```

Comment:
Does auth_timeout function as expected?

Reply:
I will check

the way it gets passed through is interesting, we pop it out to set the default if None, then pass it into CBSAuthenticator as a kwargs, only to pop it out again and compare with the default again if None

Reply:
Auth timeout checks the return time of the service over the mgmt_link when authenticating/negotiating the CBS auth. One thing is that it does change name to timeout [here when it gets passed into execute_operation](https://github.com/Azure/azure-sdk-for-python/blob/c143274b96c29bc5e17e893164222abf0098bbc5/sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/management_link.py#L203)